backend/controllers/movie.mjs

```diff
@@ -9,12 +9,12 @@ const movieCtrl = Router()
 movieCtrl.get('/', authUser , async (req,res,next) => {
     try{
         const userId = req.body.user._id;
-        const movies = await getUserAsync(userId)
-        .then(user => getMoviesAsync(user.watchList) ).catch(() => null);
-        if(!movies) throw new ErrorModel(500,"unable to get movies - server error")
+        const user = await getUserAsync(userId);
+        const movies = await getMoviesAsync(user.watchList);
+        if(!movies) throw new ErrorModel(404,"movies not found");
```

Comment:
As I see it if you don't get movies here maybe there is no movie, but maybe i'm wrong